db/1.8/modules/ROOT/pages/database-connector-examples.adoc

```diff
@@ -1,8 +1,30 @@
 = Database Examples
 :page-aliases: connectors::db/db-connector-query.adoc, connectors::db/db-connector-insert-update-delete.adoc, connectors::db/db-connector-sp.adoc, connectors::db/db-connector-ddl.adoc, connectors::db/db-connector-bulk-ops-ref.adoc, connectors::db/db-connector-datatypes-ref.adoc, connectors::db/db-connector-transactions-ref.adoc, connectors::db/db-connector-execute-script-ref.adoc, connectors::db/database-connector-examples.adoc
 
-Anypoint Connector for Database (Database Connector) examples help you configure database connections, configure connection pools, query the database, and execute stored procedures, DDL statements, and bulk operations.
-
+Anypoint Connector for Database (Database Connector) examples help you to:
+
+* <<configure-database-connection,Configure a Database Connection>> +
+Connect to the most popular databases such as Derby, Microsoft SQL Server, MySQL, Oracle, and a generic database. Additionally, configure the connector to connect to a global data source, set a JDBC driver, configure connection pooling, and connection to an Oracle database with TNS.
```

Comment:
The "Additionally, configure..." text doesn't flow well from the previous sentence. I suggest:  "Also helps you..." for the second sentences, where relevant.